test/e2e/persistentips_test.go

```diff
@@ -40,6 +40,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const networkInterfaceName = "multus"
```

Comment:
this is both the VMI interface/network logical name and the field at VMI status to look for so and is just for secondaries so let's name this 
`secondaryLogicalNetworkInterfaceName`

Reply:
fixed